src/components/toggle/Toggle.js

```diff
@@ -56,8 +62,10 @@ const Toggle = ({
                     className="toggle-simple-input"
                     disabled={isDisabled}
                     name={name}
+                    onBlur={onBlur}
                     onChange={onChange}
                     type="checkbox"
+                    value={value}
```

Comment:
Is this going to cause issues with existing usages of this component?

Reply:
I dont expect it since Toggles are controlled by the parent and rely on the checked/isOn boolean. But I can remove it for now, its dud anyways.